source/administration/config-settings.rst

```diff
@@ -2313,6 +2313,24 @@ If the "No encryption" option is selected it is highly recommended that the AD/L
 | This feature's ``config.json`` setting is ``"ConnectionSecurity": ""`` with options ``""``, ``"TLS"``, and ``"STARTTLS"``.                                           |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Private Key
+^^^^^^^^^^^^
+
+(Optional) The private key file provided by your LDAP Authentication Provider and uploaded if TLS client certificates are being used as the primary authentication mechanism.
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"PrivateKeyFile": false`` with string input.                                                                             |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Public Certificate
+^^^^^^^^^^^^^^^^^^
+
+(Optional) The public TLS certificate file provided by your LDAP Authentication Provider and uploaded if TLS client certificates are being used as the primary authentication mechanism.
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"PublicCertificateFile": false`` with with string input.                                                                 |
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+
 Skip Certificate Verification
```

Comment:
@sbishel, wondering if this needs to be updated in light of the ability to upload certificates/keys so that it references needing to be ``true`` or ``false`` if using the certificate/key?

Reply:
No, these are not actually related. Although I did notice that the existing text on line 2338 is not correct.
```(Optional) The attribute in the AD/LDAP server that will be used to populate the nickname of users in Mattermost.```

Reply:
I saw that and removed it, but then I went through some code and saw a reference to profile photo so I left it in. It wasn't added in this PR though. Thanks for confirming!

Reply:
Oh, should I remove it entirely or does it need a rewrite?